src/components/asset-list/RecyclerAssetList2/cards/CardRowWrapper.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { Box, Inset } from '@/design-system';
 
-export const CardRowWrapper = ({ children }: { children: React.ReactNode }) => {
+export const CardRowWrapper = React.memo(function CardRowWrapper({ children }: { children: React.ReactNode }) {
```

Comment:
these memos are ok to leave - even with react compiler. technically it doesn't apply memo around components since in theory they skip most inner work anyway, but it does gain just a tiny bit of performance still so ok